app/controllers/api/v1/landings/landing_themes_controller.rb

```diff
@@ -1,4 +1,5 @@
-class Api::V1::Landings::LandingThemesController < Api::V1::BaseController
+class Api::V1::Landings::LandingThemesController < Api::V1::Landings::BaseController
+  before_action :retrieve_landing
```

Comment:
```suggestion
  before_action :retrieve_landing
  
```
C'est bizarre que le linter n'ai pas voulu une ligne vide ici